bucket/lite.json

```diff
@@ -0,0 +1,19 @@
+{
+    "version": "1.11",
+    "description": "A lightweight text editor written in Lua.",
+    "homepage": "https://github.com/rxi/lite",
+    "license": "MIT",
+    "url": "https://github.com/rxi/lite/releases/download/v1.11/lite.zip",
+    "hash": "da3701cf6985bc493cb11b37556325551c7ce8606f5502ff3f15199039835eef",
+    "bin": "lite.exe",
+    "shortcuts": [
+        [
+            "lite.exe",
+            "Lite"
+        ]
+    ],
+    "checkver": "github",
```

Comment:
There should be a persist - see https://github.com/ScoopInstaller/Extras/pull/4612#pullrequestreview-472285387

Reply:
Persist `data` now